compiler/src/main/scala/dx/compiler/ExecutableCompiler.scala

```diff
@@ -44,6 +44,8 @@ object ExecutableCompiler {
     Parameter(Constants.WorkflowInputLinks, Type.TOptional(Type.THash))
   val OutputIdParameter: Parameter =
     Parameter(Constants.OutputId, Type.TString)
+  val ExtraOutputsParameter: Parameter =
```

Comment:
Shouldn't this be combined with the changes in https://github.com/dnanexus/dxCompiler/pull/214/files -- reserved parameter group? @jdidion

Reply:
They are separate changes. I'll take care of it when I merge the other PR.